src/org/jitsi/impl/neomedia/rtp/MediaStreamTrackDesc.java

```diff
@@ -150,6 +154,13 @@ void update(RawPacket pkt, FrameDesc frameDesc, long nowMs)
 
         if (!frameDesc.isIndependent())
         {
+            if (!frameIsNew)
+            {
+                // We check for stream suspension whenever we receive a new
```

Comment:
nit: maybe this is better as `We only check for stream suspension if this is the first packet of a new frame`?